lib/l10n/app_ru.arb

```
@@ -337,5 +338,7 @@
   },
   "appLogsFilteredCount": "{shown} из {total}",
   "invalidRelayUrlScheme": "URL должен начинаться с wss:// или ws://",
-  "invalidRelayUrl": "Неверный URL реле"
+  "invalidRelayUrl": "Неверный URL реле",
+  "helpAndFeedback": "Help and feedback",
```

RuboCop (1.85.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)